lib/Model/Query.js

```diff
@@ -158,7 +158,7 @@ function Query(model, collectionName, expression, options) {
   // because otherwise maybeUnload would be looping through the entire results
   // set of each query on the same collection for every doc checked
   //
-  // Map of id -> true
+  // Map of id -> count of ids
   this.idMap = {};
```

Comment:
I also noticed that subscriptions.js does a [plain property check on `idMap`](https://github.com/derbyjs/racer/blob/3d9cb6ea6d5a8326e7ad51d7bc34542ec20f0e1d/lib/Model/subscriptions.js#L194) instead of using `hasOwnProperty`, which means that the check will be incorrect if an id happens to be the name of an Object prototype property. Do you think that matters enough to fix? Racer-generated ids are UUIDs, so it's only a problem for manually-specified ids.